src/definitions/modules/progress.js

```diff
@@ -238,6 +259,7 @@ $.fn.progress = function(parameters) {
               .replace('{total}', total)
               .replace('{left}', left)
               .replace('{percent}', percent)
+              .replace('{bar}', settings.text.bars[index] || '')
```

Comment:
:thinking: ok, you already covered that by adding ` || ''` But i still think it's more consistent to use `index_` instead

Reply:
Changed to use `index_` since `index` may be undefined (not given).
However, I think fallback to empty string `''` is required if `{bar}` is undefined.
https://github.com/fomantic/Fomantic-UI/pull/582/commits/52131989fb0b8c288d66223e757c74604a6cf79a